src/include/network/lan/complex.rb

```diff
@@ -566,6 +628,8 @@ def MainDialog(init_tab)
         break if input_done?(ret)
       end
 
+      @builder = nil
```

Comment:
is it needed?

Reply:
most probably not. Just wanted to catch some unexpected accesses.